computers/paratera-th2/paratera-th2-setup.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
could you please delete the corresponding file without -setup?

Reply:
if we're moving to the `-setup` and `-configure` nomenclature now, please also rename the files in the other subfolders and adapt the README.md.

Thanks a lot!

Reply:
Sorry, my fault. A little fading out after taking cold pills :p 
 fixed.